src/Components/design/Footer/Footer.jsx

```diff
@@ -0,0 +1,16 @@
+import React from "react";
+import PropTypes from "prop-types";
+import "./Footer.css";
+
+export default function Footer(props) {
+  return (
+    <div className="footer">
+      <div className="footer-content">{props.footerText}</div>
+    </div>
```

Comment:
I recommend we use the footer tag for semantics

```suggestion
    <footer className="footer">
      <div className="footer-content">{props.footerText}</div>
    </footer>
```